scout/server/blueprints/genes/templates/genes/gene.html

```diff
@@ -82,9 +82,9 @@ <h1>
               </span>
             </li>
             <li class="list-group-item">
-              pLi Score(ExAC)
+              pLi Score(ExAC) - (compare to <a href={{ record.gnomad_link }} target="_blank">GnomAD here</a>)
```

Comment:
How often would it happen that genes don't have Ensembl Ids? Because if it's extremely rare then it's fine, otherwise the record.gnomad_link will be none

Reply:
Hm, that shouldn't really happen, but we may - as the tests suggest - not have pLI for all genes. Making it conditional on that, which should hopefully take care of all of the rare missing ENSEMBL ids as well.